docs/index.adoc

```diff
@@ -17,6 +17,7 @@ necessary info to use or contribute to SloTex NLP in your own environment.
 == Project goals
 
 == Project founders
+Find out more about the founders link:here[http://www.sklad-kadri.si/si/razvoj-kadrov/po-kreativni-poti-do-znanja-pkp/] 
```

Comment:
Please correct the adoc format for a link please use:
here[http://www.sklad-kadri.si/si/razvoj-kadrov/po-kreativni-poti-do-znanja-pkp/].

Reply:
Sorry, see the docs how to add link to adoc correctly.

Reply:
:+1: